src/Pages/Shop/Shop.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react'
+import "./Shop.css"
+import { Navbar } from '../../Components/Navbar/Navbar'
+import { Sidebar } from '../../Components/Sidebar/Sidebar'
+import { useProductAvailable } from "../../Context/product-context"
+import { ProductCard } from "../../Components/Card/ProductCard"
+import { useLocation } from "react-router-dom"
+
```

Comment:
Same you can import all files in single file

Reply:
Done.